EndlessClient/UIControls/CharacterInfoPanel.cs

```diff
@@ -168,6 +174,12 @@ protected virtual async Task DeleteButtonClick()
         protected virtual void DoUpdateLogic(GameTime gameTime)
         {
             _characterControl.Update(gameTime);
+
+            var keyboardState = _userInputProvider.CurrentKeyState;
+            if (keyboardState.IsKeyDown((Microsoft.Xna.Framework.Input.Keys)49+_characterIndex))
```

Comment:
Can you do `Keys.D1`+_characterIndex? I think C# is pretty flexible with adding values to enums in this way. 

Also, just add the using statement instead of fully qualifying the type here. FYI you can easily add the `using Microsoft.Xna.Framework.Input` via the keyboard shortcut CTRL+. in VS

Reply:
Done